osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/config/DlmsConfig.java

```diff
@@ -57,31 +60,53 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DlmsConfig.class);
 
+    @Bean(name = "protocolAdapterDlmsNettyServerBossGroup")
+    public DisposableNioEventLoopGroup serverBossGroup() {
+        // TODO - Should we pass in any parameters, like for example number of
+        // threads, to the constructor?
+        return new DisposableNioEventLoopGroup();
+    }
+
+    @Bean(name = "protocolAdapterDlmsNettyServerWorkerGroup")
+    public DisposableNioEventLoopGroup serverWorkerGroup() {
+        // TODO - Should we pass in any parameters, like for example number of
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)